cmd/controller/controller.go

```diff
@@ -24,8 +24,11 @@ var (
 	configPath    string
 	masterURL     string
 	importerImage string
-	pullPolicy    string
-	verbose       string
+	clonerImage    string
+	importerPullPolicy    string
+	clonerPullPolicy       string
+	importerVerbose       string
+	clonerVerbose          string
```

Comment:
Is there a reason to have different level of verbosity for different components of CDI?  I think the verbosity should be global to the deployment.

Reply:
as I mentioned above, this was changed to have one verbosity and pull policy for both controllers.

Reply:
Please push the changes then so reviewers can see the latest and greatest code.